src/basic/missing_syscall.h

```diff
@@ -26,9 +26,11 @@
 #include <sys/types.h>
 
 #if !HAVE_PIVOT_ROOT
-static inline int pivot_root(const char *new_root, const char *put_old) {
+static inline int _pivot_root(const char *new_root, const char *put_old) {
```

Comment:
hmm, I think we should stay away from the "_" prefix if it's easy to, and this is an excercise in namespace hygiene after all. To quote GNU:

> In addition to the names documented in this manual, reserved names include all external identifiers (global functions and variables) that begin with an underscore (‘_’) and all identifiers regardless of use that begin with either two underscores or an underscore followed by a capital letter are reserved names. This is so that the library and header files can define functions, variables, and macros for internal purposes without risk of conflict with names in user programs.

hence, why not pick any other prefix, maybe `private_`, `missing_`, `xxx_` or `local_` or so?